Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp

```diff
@@ -148,6 +149,11 @@ void CreateSchemaDatabase(TArray<UClass*> Classes)
 
 		FString FilePath = FString::Printf(TEXT("%s%s"), *PackagePath, *FPackageName::GetAssetPackageExtension());
 		bool bSuccess = UPackage::SavePackage(Package, SchemaDatabase, EObjectFlags::RF_Public | EObjectFlags::RF_Standalone, *FPackageName::LongPackageNameToFilename(PackagePath, FPackageName::GetAssetPackageExtension()));
+
+		if(!bSuccess)
+		{
+			FMessageDialog::Debugf(FText::FromString(TEXT("Unable to save Schema Database!")));
```

Comment:
Let's also include the path in here too.

Reply:
You're so god damn smart dude